flight/move_to_test.py

```diff
@@ -0,0 +1,94 @@
+"""
+Main driver code for moving drone to each waypoint
+"""
+
+import asyncio
+import goto
+from mavsdk import System
+import logging
+from typing import List
+import sys
+import extract_gps
+
+
+async def run() -> None:
+    """
+    This function is a driver to test the goto function and runs through the
+    given waypoints in the lats and longs lists at the altitude of 100.
+    Makes the drone move to each location in the lats and longs arrays at the altitude of 100 and
+    Notes
+    -----
+    Currently has 3 values in each the Lats and Longs array and code is looped and will stay in that loop
+    until the drone has reached each of locations specified by the latitude and longitude and
+    continues to run until forced disconnect
+    """
+    # Put all latitudes, longitudes and altitudes into seperate arrays
+    lats: List[float] = []
+    longs: List[float] = []
+    altitudes: List[float] = []
+
+    waypoint_data = extract_gps.extract_gps("./data/waypoint_data.json")
+    waypoints = waypoint_data["waypoints"]
+    for i in waypoints:
+        lats.append(i.latitude)
+        longs.append(i.longitude)
+        altitudes.append(i.altitude)
+    # print(lats)
+    # print(longs)
+    # print(altitudes)
```

Comment:
You can clean up the file a bit by getting rid of these commented print statements.